TimeTracker/MainWindow.xaml.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Windows;
+using System.Windows.Input;
+using System.Windows.Threading;
+
+namespace TimeTracker
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+
+    public partial class MainWindow : Window
+    {
+        private DateTime _StartTime;
+        private DateTime _EndTime;
+        private TimeSpan _TimeDifference;
+        private bool _IsReset;
+        private string _TimerDescription;
+
+        private DispatcherTimer DispatcherTimer { get; }
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            DispatcherTimer = new();
+            DispatcherTimer.Interval = new TimeSpan(1);
+            DispatcherTimer.Tick += DispatcherTimer_Tick;
+        }
+
+        private void DispatcherTimer_Tick(object sender, EventArgs e)

```

Comment:
The nullable signature of the event handler expects the `sender` to be declared nullable.
```suggestion
        private void DispatcherTimer_Tick(object? sender, EventArgs e)

```